.prettierignore

```diff
@@ -7,3 +7,5 @@ TEST_BUILD_OUT
 web_modules
 test/create-snowpack-app/test-install
 .snowpack
+
+!snowpack/**
```

Comment:
I'd love to avoid adding `!snowpack/**`  here if possible. What about replacing `build` with:

```diff
- build
+ test/**/build
+ create-snowpack-app/**/build
```